io.crow_translate.CrowTranslate.yaml

```diff
@@ -0,0 +1,91 @@
+id: org.kde.CrowTranslate
+sdk: org.kde.Sdk
+runtime: org.kde.Platform
+runtime-version: 5.15-23.08
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  # The app supports capturing screen via KWin and Mutter DBus APIs, but
+  # access to it seems overly broad in spite of the app also supporting XDP API
+  #- --talk-name=org.kde.KWin
+  #- --talk-name=org.gnome.Shell
+rename-icon: crow-translate
+copy-icon: true
+command: crow
+cleanup:
+  - '*.a'
+  - '*.la'
+  - /include
+  - /lib/cmake
+  - /lib/pkgconfig
+modules:
+  - name: crow-translate
+    buildsystem: cmake-ninja
+    sources:
+      - type: git
+        url: https://github.com/crow-translate/crow-translate.git
+        tag: 2.11.1
+        commit: 7113ff01a994584ed0a3ec557fc3a694fcb06b63
+        x-checker-data:
+          type: git
+    modules:
+      - name: tesseract
+        buildsystem: cmake-ninja
+        builddir: true
+        config-opts:
+          - -DBUILD_SHARED_LIBS:BOOL=ON
+        cleanup:
+          - /bin
+        sources:
+          - type: git
+            url: https://github.com/tesseract-ocr/tesseract.git
+            tag: 5.3.4
+            commit: 8ee020e14cf5be4e3f0e9beb09b6b050a1871854
```

Comment:
Please use release tarballs as much as possible, not direct Git repos.

Reply:
Switched to tarballs. I just used previously submitted yaml.